script.xbmc.unpausejumpback/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.xbmc.unpausejumpback" name="Unpause Jumpback" version="3.0.0" provider-name="bossanova808,Memphiz,Lucleonhart,schumi2004,dkadioglu">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.service" library="default.py"></extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Kodi automatic jump back on un-pausing of videos</summary>
+    <summary lang="bg">Превърта назад след пауза</summary>
```

Comment:
this should be bg_GB

Reply:
Happy to change it of course, but just for my understanding... I always assumed that en_GB was for English, with a 'class' of British English essentially.  Is there an equivalent for Bulgarian then?

Reply:
Ah I think I see - maybe `bg_BG`?  not `bg_GB` ?